controls/cis_rhel7.yml

```diff
@@ -1226,3 +1226,322 @@ controls:
+    - l2_workstation
+    automated: yes
+    rules:
+    - auditd_data_retention_max_log_file
+    - var_auditd_max_log_file=6
+
+  - id: 4.1.2.2
+    title: Ensure audit logs are not automatically deleted (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - auditd_data_retention_max_log_file_action
+    - var_auditd_max_log_file_action=keep_logs
+
+  - id: 4.1.2.3
+    title: Ensure system is disabled when audit logs are full (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - auditd_data_retention_space_left_action
+    - var_auditd_space_left_action=email
+    - auditd_data_retention_action_mail_acct
+    - var_auditd_action_mail_acct=root
+    - auditd_data_retention_admin_space_left_action
+    - var_auditd_admin_space_left_action=halt
+
+  - id: 4.1.2.4
+    title: Ensure audit_backlog_limit is sufficient (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    notes: <-
+      Note that currently the value is hardcoded to 8192
+    rules:
+    - grub2_audit_backlog_limit_argument
+
+  - id: 4.1.3
+    title: Ensure events that modify date and time information are collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_time_adjtimex
+    - audit_rules_time_clock_settime
+    - audit_rules_time_settimeofday
+    - audit_rules_time_stime
+    - audit_rules_time_watch_localtime
+
+  - id: 4.1.4
+    title: Ensure events that modify user/group information are collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_usergroup_modification_group
+    - audit_rules_usergroup_modification_gshadow
+    - audit_rules_usergroup_modification_opasswd
+    - audit_rules_usergroup_modification_passwd
+    - audit_rules_usergroup_modification_shadow
+
+  - id: 4.1.5
+    title: Ensure events that modify the system's network environment are collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_networkconfig_modification
+
+  - id: 4.1.6
+    title: Ensure events that modify the system's Mandatory Access Controls are collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: partially # rule for checking audit watch on /usr/share/selinux is missing
+    rules:
+    - audit_rules_mac_modification
+
+  - id: 4.1.7
+    title: Ensure login and logout events are collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_login_events_faillock
+    - audit_rules_login_events_lastlog
+
+  - id: 4.1.8
+    title: Ensure session initiation information is collected (Automated)
+    levels:
+    - l2_server
+    - l2_workstation
+    automated: yes
+    rules:
+    - audit_rules_session_events_wtmp
+    - audit_rules_session_events
```

Comment:
It seems to me that `audit_rules_session_events` covers `utmp`, `btmp` and `wtmp`.
And· there are also individual rules for each path, for example `audit_rules_session_events_utmp`

I suggest to go one approach or the other, but not overlap coverage.